fe1-web/src/core/components/PoPTouchableOpacity.tsx

```diff
@@ -1,19 +1,19 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import { Pressable, StyleProp, View, ViewStyle } from 'react-native';
-import { TouchableOpacity } from 'react-native-gesture-handler';
+import { StyleProp, TouchableOpacity, View, ViewStyle } from 'react-native';
 
 import { ExtendType } from 'core/types';
-
 /**
  * Wraps a touchable opacity in a pressable component to make karate tests work :)
+ * /!\ no longer wraps due to UI bugs (dayan9265, 06-06-2023)
```

Comment:
Should be nice if it is mention the state regarding Karate (Was this changed tested, or not)

Reply:
I asked for the karate tests, and the response was that currently karate tests are only for the back-ends (so not tested, but no impact currently)

Reply:
If you have any reference (like a GitHub issue link) to put there, I think it could be nice to explain the issue.